pages/create-study.js

```diff
@@ -60,18 +63,40 @@ export default function CreateStudy() {
         error={!isValidParticipantLimit}
         helperText={isValidParticipantLimit ? '' : 'Must be between 1 and 500'}
       />
-      <TextField
-        id="outlined-basic"
-        label="Input profile"
-        variant="outlined"
-        type="number"
-        value={inputProfile}
-        onChange={(e) => setInputProfile(e.target.value)}
-        sx={{ mb: 4 }}
-        inputProps={{ min: 0, max: 16 }}
-        error={!isValidInputProfile}
-        helperText={isValidInputProfile ? '' : 'Must be between 0 and 16'}
-      />
+      <FormControl style={{minWidth: 120}}>
+        <InputLabel id="outlined-basic">Input Profile</InputLabel>
+        <Select
+          labelId="outlined-basic-label"
+          id="outlined-basic"
+          type="number"
+          defaultValue={0}
+          value={inputProfile}
+          label="Input Profile"
+          onChange={(e) => setInputProfile(e.target.value)}
+          sx={{ mb: 4 }}
+          inputProps={{ min: 0, max: 16 }}
+          error={!isValidInputProfile}
+          helperText={isValidInputProfile ? '' : 'Must be between 0 and 16'}
+        >
+          <MenuItem value={0}>0</MenuItem>
+          <MenuItem value={1}>1</MenuItem>
+          <MenuItem value={2}>2</MenuItem>
+          <MenuItem value={3}>3</MenuItem>
+          <MenuItem value={4}>4</MenuItem>
+          <MenuItem value={5}>5</MenuItem>
+          <MenuItem value={6}>6</MenuItem>
+          <MenuItem value={7}>7</MenuItem>
+          <MenuItem value={8}>8</MenuItem>
+          <MenuItem value={9}>9</MenuItem>
+          <MenuItem value={10}>10</MenuItem>
+          <MenuItem value={11}>11</MenuItem>
+          <MenuItem value={12}>12</MenuItem>
+          <MenuItem value={13}>13</MenuItem>
+          <MenuItem value={14}>14</MenuItem>
+          <MenuItem value={15}>15</MenuItem>
+          <MenuItem value={16}>16</MenuItem>

```

Comment:
thinking we should make the displayed values of these abbreviated descriptions instead of just the enum/int value i.e. 
H. Slider
V. Slider
Joystick
Toggle
H. Slider + V. Slider
...
Eventually I think we could also try using thumbnails/images of the controls (once we have the controls implemented)